pipeline/vars/common.groovy

```diff
@@ -237,4 +266,68 @@ def sendUMBMessage(def msgType){
     return sendResult
 }
 
+def fetchTier1Compose(){
+    /*
+       Fetch the compose to be tested for tier1, is there is one
+    */
+    def defaultFileDir = "/ceph/cephci-jenkins/latest-rhceph-container-info"
+    def tier0Json = "${defaultFileDir}/RHCEPH-${env.rhcephVersion}-tier0.json"
+    def tier1Json = "${defaultFileDir}/RHCEPH-${env.rhcephVersion}-tier1.json"
+    def tier1FileExists = sh(returnStdout: true, script: """if [ -f '${tier1Json}' ] ; then echo 'yes' ; else echo 'no' ; fi""").trim()
```

Comment:
Checking the returnCode of `ls -l ${tier1Json}` could be sufficient.

Reply:
Changed the code accordingly